eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionWriterTest.java

```diff
@@ -40,8 +38,8 @@
 import static org.junit.Assert.assertTrue;
 
 @RunWith(SpringRunner.class)
-@DataJpaTest
-@ContextConfiguration(classes = {SubmittedVariantAccessioningConfiguration.class, InputParametersConfiguration.class})
+@AutoConfigureDataJpa
```

Comment:
For other reviewers, I was wondering about this change and then noticed that [@DataJpaTest](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/test/autoconfigure/orm/jpa/DataJpaTest.html) involves what `@AutoConfigureDataJpa` does and quite a lot more.